tests/unittests/tests-fib/tests-fib.c

```diff
@@ -27,6 +27,14 @@ static fib_table_t test_fib_table = { .data.entries = _entries,
                                       .mtx_access = MUTEX_INIT,
                                       .notify_rp_pos = 0 };
 
+/*
+* @brief helper to create FIB test addresses
+*/
+static void _set_FIB_test_addr(char * addr_dst, size_t add_buf_size, size_t i) {
```

Comment:
```suggestion
static void _set_fib_test_addr(char * addr_dst, size_t add_buf_size, size_t i) {
```

uppercase is reserved for macros per our coding convention.

But at this point, why not stick with `%u` and a cast to `unsigned` here? After all, the numbers have to be < 100 anyway.

Or maybe even go for

```
    for (unsigned i = 0; i < entries, i++) {
        /* construct "addresses" for the FIB */
        snprintf(addr_dst, add_buf_size, "Test address %02u", i);
        ...
```

Reply:
My first version replaced all `size_t` types with `unsigned`, because the values are always below 100. But then I thought, maybe I shouldn't make so many changes not related to the PR. 
But if desired, I can do that and also change the other `_FIB_` function names.

Can I force push a version with a shorter commit message (e.g., just `unittests/tests-fib: test cleanup`) to get rid of the "Commit message too long" error? I could also autosquash while I'm at it.

Reply:
Ah, I was unaware of the surrounding code. It is fine to keep the name consistent with the other `..._FIB_...` names. 

I agree that fixing the code style of unit tests is out of scope of the PR.

Reply:
Seeing that you did fix the code file: No need to revert. It's also fine to fix that.